src/BloomBrowserUI/bookEdit/js/synphonyApi.ts

```diff
@@ -27,13 +27,13 @@ class SynphonyApi {
       lang_data.addGrapheme(this.source.letters.split(' '));
       lang_data.addWord(this.source.moreWords.split(' '));
       lang_data.LanguageSortOrder = this.source.letters.split(' ');
+    }
```

Comment:
Very puzzling. We seem to have added a closing brace, with nothing to match it. That would imply that either the old or new code was syntactically wrong.
Apparently we are now going to set up the stages even if there are no letters. That makes sense for word-list mode, but will this re-break the pathological case where they ARE using letters but haven't configured any?